src/components/Reports/GoalCalculator/CalculatorSettings/Steps/SpecialIncomeStep/SpecialIncomeStep.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useContext } from 'react';
+import { Box, Container, Typography, styled } from '@mui/material';
+import { Form, Formik, FormikProps } from 'formik';
+import { useTranslation } from 'react-i18next';
+import * as Yup from 'yup';
```

Comment:
In the other components, we use `yup` instead of `Yup`. There are a few other imports that could be changed.

```suggestion
import * as yup from 'yup';
```